server/api/main.py

```diff
@@ -533,6 +537,21 @@ def _start_periodic_pagination_cache_monitoring():
         )
 
 
+def _start_periodic_project_summaries_calculation():
+    interval = int(config.monitoring.projects.summaries.cache_interval)
+    if interval > 0:
+        logger.info(
+            "Starting periodic project summaries calculation", interval=interval
+        )
+        run_function_periodically(
+            interval,
+            server.api.crud.projects.Projects().refresh_project_resources_counters_cache.__name__,
+            False,
+            server.api.db.session.run_async_function_with_new_db_session,
```

Comment:
Maybe it is a gap with my understanding, but why we need to run this function with new db session each time?